automation/terraform/monitoring/README.md

```diff
@@ -0,0 +1,180 @@
+<p><img src="https://storage.googleapis.com/coda-charts/Mina_Icon_Secondary_RGB_Black.png" alt="Mina logo" title="mina" align="left" height="60" /></p>
+
+# Testnet Monitoring & Alerting :fire_engine: Guide
+
+**Table of Contents**
+- [Updating Testnet Alerts](#update-testnet-alerts)
+    - [Developing alert expressions](#developling)
+    - [Testing](#testing)
+    - [Deployment](#deployment)
+- [Alert Status](#alert-status)
+    - [GrafanaCloud Config](#grafancloud-config)
+    - [Alertmanager UI](#alertmanager-ui)
+    - [PagerDuty](#pagerduty)
+- [HowTo](#howto)
+    - [Silence Alerts](#silence-alerts)
+    - [Update Alert Receivers](#update-alert-receivers)
+    - [View Alert Metrics](#view-alert-metrics)
+
+## Updating Testnet Alerts
+
+#### Developing alert expressions
+
+Developing alert expressions consists of using Prometheus's domain-specific [query language](https://prometheus.io/docs/prometheus/latest/querying/basics/) ([examples](https://prometheus.io/docs/prometheus/latest/querying/examples/)) coupled with its alert rules specification [format](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) for devising metric and alerting conditions/rules.
+
+To enable variability when defining these rules, each rule set or group is implemented using *terraform*'s [template_file](https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file) and `(${ ... })` templating mechanisms for including variable substitution where appropriate (**note:** variable substitutions are *optional* and provided as defaults - alert rules can be defined completely according to a custom specification).
+
+A standard set of such testnet alert rules based on templates is defined [here](https://github.com/MinaProtocol/mina/blob/develop/automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl#L6) and should be edited when adding rules to be applied to all testnets.
```

Comment:
```suggestion
A standard set of such testnet alert rules based on templates is defined [here](https://github.com/MinaProtocol/mina/blob/develop/automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl#L6) and must be edited when you add rules to be applied to all testnets.
```

Reply:
should is ambiguous, let's use "must" if we mean "must"
see https://developers.google.com/style/word-list#should